pineappl_py/src/subgrid.rs

```diff
@@ -142,6 +143,53 @@ impl PySubgridParams {
     }
 }
 
+/// PyO3 wrapper to :rustdoc:`pineappl::subgrid::Mu2 <subgrid/struct.Mu2.html>`
+#[pyclass]
+#[repr(transparent)]
+pub struct PyMu2 {
+    pub(crate) mu2: Mu2,
+}
+impl PyMu2 {
+    pub(crate) fn new(mu2: Mu2) -> Self {
```

Comment:
Drop this block ...

Reply:
[e557df8](https://github.com/NNPDF/pineappl/pull/222/commits/e557df89a81f2a1b225deeae42df854955d3bc3c)